mobile/src/screens/journal/journal.tsx

```diff
@@ -42,6 +46,18 @@ const Journal: React.FC = () => {
     // TODO: Implement actual functionality for the onPress event
   }, []);
 
+  const handleShowModal = (): void => {
+    setIsModalVisible(true);
+  };
+
+  const hanleCloseModal = (): void => {
+    setIsModalVisible(false);
+  };
+
+  const handleDeleteNote = (): void => {
+    setIsModalVisible(false);
+  };
```

Comment:
There're actually two same functions) Correct me if they'll be different later

Reply:
@Dmytro-Kucherenko its different functions, because their functionality is not ready for now. its draft PR